dotcom-rendering/src/web/components/ArticleTitle.tsx

```diff
@@ -57,6 +60,15 @@ const immersiveMargins = css`
 	}
 `;
 
+const livePulseIconStyles = (palette: Palette) => css`
```

Comment:
Could these styles live inside the `PulsingDot` component?

Reply:
Yes, please!